tests/managedfolder_tests.py

```diff
@@ -57,12 +57,12 @@ def updload_download_test():
 
     count = len(managedfolder.list_contents()['items'])
     with open(stuff, "r") as f:
-        managedfolder.put_file('stuff', f)
+        managedfolder.put_file('subfolder/stuff', f)
```

Comment:
These test files are un-used and have not been updated since 5+ years :-) 

In order to test your modification, you need to add a test in the main DSS repository (along with other integration tests). This is how we currently these the Python client

Note: we should open a PR to remove these test files because you're not the first person to get confused by them

Reply:
Also, this test could detect more errors if it was adding special characters (accents, percents, ...), since it is a frequent problem with Python 2/3

Reply:
FYI I remove these tests in https://github.com/dataiku/dataiku-api-client-python/commit/f9fb9881fd1db591bfc29a2e41ca65dc941318a5 so that people don't try to update them in the future